generate-github-io

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
I like this - the github.io page makes linking to/navigating the documentation really easy. I couldn't find an easy link to it, though - I would add a link in the README of the repo.